tests/algorithm/scripts/send_test_records.py

```diff
@@ -0,0 +1,66 @@
+import csv
+
+import requests
+from helpers import dict_to_pii
+
+
+def send_test_records(test_csv, algorithm_name, api_url):
+    output_data = []
+
+    print("Sending test records to the API...")
+
+    # Read the CSV file using the csv module
+    with open(test_csv, mode='r', newline='', encoding='utf-8') as file:
+        reader = csv.DictReader(file)
+
+        for row in reader:
+            # Replace None and "NaN" values with an empty string
+            record_data = {k: ("" if v in [None, "NaN"] else v) for k, v in row.items()}
+
+            # Get info from row
+            match_info = {
+                "test_case_number": record_data['Test Case #'],
+                "match_id": record_data['Match Id'],
+                "should_match": record_data['Expected Result'],
+            }
+
+            pii_record = dict_to_pii(record_data)
+            
+            response = send_record(pii_record, algorithm_name, api_url)
+
+            if response:
+                output_row = {
+                    "Test Case #": match_info['test_case_number'],
+                    "Expected Result": match_info['should_match'],
+                    "Match Result": response['prediction'],
```

Comment:
Our valid `prediction` values of `"match"`, `"no_match"`, and `"possible_match"` don't exactly align with those in the `Expected Result` column of the NBS test cases. Is there transformation somewhere, or are we expecting our valid values in this column? If the latter, is validation somewhere needed?

For context, having the same set of values in the `Expected Result` and `Match Result` columns of the `output.csv` is very helpful for calculating match accuracy performance metrics down the line (e.g., False Positives, False Negatives, Sensitivity, Specificity, etc.). If there are transformations to results needed, I think that should live somewhere in this testing framework and outside of the calculations for these metrics.

E.g.,`"Should Be a Match"` in NBS test cases vs. `"match"` in Record Linkage algorithm prediction, `"Should Be a Match?"` vs. `"possible_match"`, etc.